### ISSUE TITLE
Linter reports an error in Angular templates when using bound template syntax

### ISSUE THREAD

**Issue by ahmadalfy** [dequelabs/axe-core#3383]
Product: I think it's axe Linter

Expectation: The linter reports when an image is missing an `alt` attribute as expected. In Angular, you could use a syntax to bind the value of the attribute to something dynamic provided from the component like the following (see `alt` between the square brackets):

```html
<img src="..." [alt]="someVariable" />
```

Actual: The linter reports this as a problem.

Motivation: I would like the linter to ignore that problem. I understand that it's out of the scope of the linter, I would love to know if there is a way to do it and I will contribute to that.

**Comment by straker** [comment]
Thanks for the issue. I'll pass along the request to the appropriate team.

**Comment by straker** [comment]
An update on this. Axe-linter 4.4.0 should have a fix for this. Please let us know if it's still not working.

**Comment by straker** [comment]
Going to close this issue as fixed. Please let us know if that is not the case.

**Comment by ahmadalfy** [comment]
@straker I am unable to verify that because I am now getting this error when I start VSCode

<img width="579" alt="image" src="https://user-images.githubusercontent.com/87611/183634375-b7a1e7b3-da40-4865-aeab-6ebc5629a194.png">

with the following output

```
[Error - 1:12:48 PM] Starting client failed
Launching server using command /Users/**/.vscode/extensions/deque-systems.vscode-axe-linter-4.4.1/dist/bin/LSP-server-4.4.1 failed.
```

I know there is a [thread](https://github.com/dequelabs/axe-core/issues/3475) to discuss that issue here, apparently something is wrong. If there's any information I can provide, I will be glad to do so
